overlay/usr/lib/inithooks/bin/opencart.py

```diff
@@ -12,6 +12,7 @@
 import re
 import sys
 import getopt
+from passlib.apps import phpass_context
 from libinithooks import inithooks_cache
 import time
 import hashlib
```

Comment:
If hashlib is no longer being used (as appears to be the case), it can be removed.